### DISCUSSION TITLE
"systemd Services" is Missing after update.

### DISCUSSION THREAD

**Discussion by kromoxpro** [netdata/netdata#13598] [Q&A]
I had version 1.35.1 and when updating netdata to version 1.36.1 the "systemd Services" option has disappeared.

I don't think the problem is the version because on other computers I also have 1.36.1 and I have the "systemd Services" option activated.

I've read the documentation for this [URL](https://learn.netdata.cloud/docs/agent/collectors/cgroups.plugin/#monitoring-systemd-services) and still can't get it to show up.

I've checked the related issues and still can't fix the problem.

Thanks.

**Comment by ilyam8** [comment]
Hi, @kromoxpro. This is a duplicate of this [discussion](https://github.com/netdata/netdata/discussions/13592).
